README.rst

```diff
@@ -1,6 +1,7 @@
 tractor
 =======
 A `structured concurrent`_, async-native "`actor model`_" built on trio_ and multiprocessing_.
```

Comment:
This looks pretty good to me. If I were you, I'd want to decouple my pitch for tractor from `multiprocessing`, and leave that as an implementation detail.

Reply:
@ryanhiebert it links to wikipedia not the stdlib; so it's in reference to actual muli-processing as an OS usage pattern.

Not sure if that was clear. Maybe *multi-processing* (with the dash) makes that more clear?

If you just mean that not focusing on it being implemented with multiple processes (since in theory we could somehow move to a threading system eventually?) then definitely worth considering.

Reply:
Oh, I was thinking it was about the stdlib module. The dash would make more sense to me.

Reply:
Yeah, the plan is to move entirely off the stdlib module asap.

The only thing blocking that is #146 for which I'd like to keep the stdlib stuff to compare against.